content/influxdb/cloud/account-management/limits.md

```diff
@@ -101,4 +101,4 @@ The following API error responses occur when your plan's service quotas are exce
 | :-----------------------------  | :-----------------------------------------  | :----------- |
 | `HTTP 413 “Request Too Large”`  | cannot read data: points in batch is too large | If a **write** request exceeds the maximum [global limit](/influxdb/cloud/account-management/limits/#global-limits) |  
 | `HTTP 429 “Too Many Requests”`  | Retry-After: xxx (seconds to wait before retrying the request) | If a **read** or **write** request exceeds your plan's [adjustable service quotas](/influxdb/cloud/account-management/limits/#adjustable-service-quotas) or if a **delete** request exceeds the maximum [global limit](/influxdb/cloud/account-management/limits/#global-limits) |
-| `HTTP 503 “Service unavailable“` | Series cardinality exceeds your plan's service quota | If **series cardinality** exceeds your plan's [adjustable service quotas](/influxdb/cloud/account-management/limits/#adjustable-service-quotas) |
+| `HTTP 429 “Service unavailable“` | Series cardinality exceeds your plan's service quota | If **series cardinality** exceeds your plan's [adjustable service quotas](/influxdb/cloud/account-management/limits/#adjustable-service-quotas) |
```

Comment:
429 is also `“Too Many Requests”`

Reply:
@pierwill Does `429` really return a "Service unavailable" message in this case? If so, that's a bug as "Too many requests" is the standard HTTP status message for `429`. If not, then we should add the "Series cardinality..." message to the previous `429` row.

Reply:
> @pierwill Does `429` really return a "Service unavailable" message in this case? If so, that's a bug as "Too many requests" is the standard HTTP status message for `429`. If not, then we should add the "Series cardinality..." message to the previous `429` row.

Just realized I was saying the same thing as @philjb.